modules/environment/aws/code-services/main.tf

```diff
@@ -316,7 +316,14 @@ resource "aws_iam_policy" "event_mapper_role_policy" {
       ],
       "Effect": "Allow",
       "Resource": "*"
-    }
+    },
+	{
+	  "Action": [
+	    "codecommit:GetCommit"
+	  ],
+	  "Effect": "Allow",
+	  "Resource": "*"
+	}
```

Comment:
can you fix the indentation so this json object aligns with the one before it?

Reply:
My bad. Just updated it!